### ISSUE TITLE
Import location as YAML file

### ISSUE THREAD

**Issue by scls19fr** [BEMServer/bemserver-ui#128]
Hello,

In BEMServer, locations have a hierachical form.
Maybe supporting YAML import should be considered as it can be a very convenient maner to populate database with sample data.

```yaml
---
sites:
- name: Site1
  latitude: -90
  longitude: -180
  description: AAAAAA
  ifc_id: AAAAAA
  buildings:
  - name: B1
    description: AAAAAA
    ifc_id: AAAAAA
    storeys:
    - name: R0
      description: AAAAAA
      ifc_id: AAAAAA
      spaces:
      - name: ROOM1
        description: AAAAAA
        ifc_id: AAAAAA
        storey_id: 0
      - name: ROOM2
        description: AAAAAA
        ifc_id: AAAAAA
        storey_id: 0
    - name: R1
      description: AAAAAA
      ifc_id: AAAAAA
  - name: B2
    description: AAAAAA
    ifc_id: AAAAAA
```

Linting and viewing such a file can be done using online tools such as https://jsonformatter.org/yaml-viewer

Kind regards

**Comment by lafrech** [comment]
This makes sense. I like the YAML representation.

Our workflow involves using a spreadsheet as an exchange format between actors of the project, and I don't think a YAML file would work because most are more comfortable using excel than a plain text editor. (Which is a pity because generating a proper comma-separated UTF-8 encoded .csv file from Excel is just a nightmare, but LibreOffice does better.)

In short we wouldn't object to adding this feature, even if it adds a dependency to pyyaml, but we won't do it.

Let's keep this open to keep track of the idea. Anyone, fell free to propose an implementation.

Transfering this to bemserver-core as this is where it happens (in `input_output`).
